task/sql-tasks.js

```diff
@@ -177,7 +279,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT 
+            count(*) AS TotalOfCurrentProducts,
+            SUM(Discontinued) AS TotalOfDiscontinuedProducts

```

Comment:
use count when you need to count recrods to show intent explicitly

Reply:
but even better to write this query like this

```
select
(select count()...) as 'T'
(select count()...) as 'TT'
```

notice that from-clause is optional